csm_web/frontend/src/components/resource_aggregation/ResourceEdit.tsx

```diff
@@ -666,40 +667,53 @@ export const ResourceEdit = ({ resource, onChange, onSubmit, onCancel }: Resourc
             <div id="resourceEditInner">
               <div className="resourceInfoEdit">
                 <div className="resourceEditHeadItem">Week Number</div>
-                <input
-                  type="text"
-                  defaultValue={resource.weekNum}
-                  placeholder="Week Number"
-                  onChange={e => onChange(e, "weekNum")}
-                  onBlur={() => handleBlur("weekNum")}
-                />
+                <div>
+                  <input
+                    type="text"
+                    defaultValue={resource.weekNum}
+                    placeholder="Week Number"
+                    onChange={e => onChange(e, "weekNum")}
+                    onBlur={() => handleBlur("weekNum")}
+                  />
+                </div>
```

Comment:
Are these divs necessary? They don't seem to be doing anything style-wise.

Reply:
The tooltip wrapper shifts the entire input down slightly, so we decided the best way to fix this is to add divs to the other two inputs to shift them by the same amount. Is there a better way to do this?

Reply:
Seems like it's because of this CSS attribute:
https://github.com/csmberkeley/csm_web/blob/fd19c62878643615a83dbbd2d4c1f9c2a15b4461/csm_web/frontend/static/frontend/css/resource_aggregation.css#L229-L231
I'd recommend changing this to instead list out exactly all of the classes that are direct div children of `.resourceInfoEdit`; i.e. I believe it's just `.resourceEditHeadItem` and `.resourceValidationError`, but double check to make sure.

This should mean that you won't need additional divs for the other elements.